plenum/server/node.py

```diff
@@ -2055,6 +2075,16 @@ def process_query(self, request: Request, frm: str):
         result = handler.get_query_response(request)
         self.transmitToClient(Reply(result), frm)
 
+    def process_action(self, request, frm):
+        # Process a execute action request
```

Comment:
At first we must perform static validation of the request and send REQACK or REQNACK to the sender depending on the result.

Reply:
We already do it in node:doStaticValidation() in line 1960 : req_handler.doStaticValidation(request)